.config_files.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl" ?>
+
+<entry_id>
+
+  <!-- This is the same as the default entry in
+       cime/config/cesm/config_files.xml except for the value for clm:
+       In a standalone clm checkout, COMP_ROOT_DIR_LND is $SRCROOT
+       rather than $SRCROOT/components/clm.
+
+       However, because of the way overrides are handled, we need to
+       re-specify the full information here rather than just overriding
+       the value for clm.
+  -->
+  <entry id="COMP_ROOT_DIR_LND">
```

Comment:
I was hoping that I could get away with just specifying the new `COMP_ROOT_DIR_LND` for clm, without duplicating all of the metadata and without duplicating the entries for `dlnd`, `slnd` or `xlnd`. Alas, this doesn't seem possible:

- When I didn't duplicate all of the metadata (group, file, etc.), the python complained about missing `file`

- When I didn't specify entries for `dlnd`, `slnd` and `xlnd`, I wasn't able to run a T compset (CISM forced by dlnd). I know it's kind of funny to run a compset without CLM from a standalone CLM checkout, but I sometimes do that for testing, and I can see some value in allowing it for scientists as well, so I don't want to lose that ability.

This isn't a big deal, but @bandre-ucar @jedwards4b - am I missing something that would allow me to make this override file briefer?

Reply:
You aren't missing anything, the override needs to be a complete entry.   I'm not aware of a way to make it shorter.

Reply:
Okay, thanks a lot for taking a look at this